projects/storefrontapp/project.json

```
@@ -141,32 +157,59 @@
           "namedChunks": false,
           "aot": true,
           "extractLicenses": true,
-          "vendorChunk": false,
-          "buildOptimizer": true,
           "serviceWorker": false,
           "tsConfig": "projects/storefrontapp/tsconfig.app.prod.json"
         },
         "development": {
-          "buildOptimizer": false,
           "optimization": false,
-          "vendorChunk": true,
           "extractLicenses": false,
-          "sourceMap": true,
-          "namedChunks": true
+          "sourceMap": true
+        },
+        "noSsr": {
+          "ssr": false,
+          "prerender": false
+        },
+        "withPrerender": {
+          "prerender": {
+            "discoverRoutes": false,
+            "routesFile": "projects/storefrontapp/prerender.txt"
+          }
         }
-      }
+      },
+      "defaultConfiguration": "production"
     },
-    "serve": {
-      "executor": "@angular-builders/custom-webpack:dev-server",
-      "options": {
-        "buildTarget": "storefrontapp:build"
+    "build-csr": {
+      "executor": "@angular-devkit/architect:concat",
+      "configurations": {
+        "development": {
+          "targets": [{ "target": "storefrontapp:build:development,noSsr" }]
+        },
+        "production": {
+          "targets": [{ "target": "storefrontapp:build:production,noSsr" }]
+        }
       },
+      "defaultConfiguration": "production"
+    },
+    "prerender": {
+      "executor": "@angular-devkit/architect:concat",
       "configurations": {
+        "development": {
+          "targets": [{ "target": "storefrontapp:build:development,noSsr" }]
+        },
         "production": {
-          "buildTarget": "storefrontapp:build:production"
+          "targets": [{ "target": "storefrontapp:build:production,noSsr" }]
+        }
+      },
+      "defaultConfiguration": "production"
```

typo (I guess due to a copy-paste): `noSsr` vs `withPrerender`
```suggestion
    "prerender": {
      "executor": "@angular-devkit/architect:concat",
      "configurations": {
        "development": {
          "targets": [{ "target": "storefrontapp:build:development,withPrerender" }]
        },
        "production": {
          "targets": [{ "target": "storefrontapp:build:production,withPrerender" }]
        }
      },
      "defaultConfiguration": "production"
```

---

Yes, you are right.